server/re/src/irods_re_serialization.cpp

```diff
@@ -1199,6 +1199,29 @@ namespace irods::re_serialization
         }
     } // serialize_vector_of_strings_ptr
 
+    static irods::error serialize_structFileExtAndRegInp_ptr(boost::any _p, serialized_parameter_t& _out)
+    {
+        try {
+            log_re::trace(__func__);
+            const auto* casted_ptr = boost::any_cast<structFileExtAndRegInp_t*>(_p);
+            _out["objPath"] = casted_ptr->objPath;
+            _out["collection_path"] = casted_ptr->collection;
+            _out["oprType"] = boost::lexical_cast<std::string>(casted_ptr->oprType);
+            _out["flags"] = boost::lexical_cast<std::string>(casted_ptr->flags);
```

Comment:
clang-tidy says `std::to_string` is preferred to `boost::lexical_cast<std::string>`. I think I agree...

Reply:
if std works, seems good.